src/main/api/2/studio-api-2.yaml

```
@@ -1115,6 +1119,159 @@ paths:
+                type: object
+                properties:
+                  result:
+                    type: object
+                    properties:
+                      response:
+                        $ref: '#/components/schemas/ApiResponse'
+                      entities:
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/ClusterMember'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    patch:
+      tags:
+        - cluster
+      summary: Update (upsert) cluster members
+      description: 'Required Permission: "UPDATE_CLUSTER"'
+      operationId: updateClusterMembers
+      requestBody:
+        description: Cluster members to update (upsert)
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                clusterMembers:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/ClusterMember'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  result:
+                    type: object
+                    properties:
+                      response:
+                        $ref: '#/components/schemas/ApiResponse'
+                      entities:
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/ClusterMember'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
```

I don't think we need to throw this error?